release-notes/opensearch-security.release-notes-2.3.0.0.md

```diff
@@ -0,0 +1,15 @@
+## 2022-08-15 Version 2.3.0.0
```

Comment:
Missed this is initial review, this should be `2022-09-15`. I'll create a PR to update.